src/content/docs/logs/ui-data/query-time-parsing.mdx

```diff
@@ -0,0 +1,419 @@
+            </td>
+            <td>
+                Best for doing quick queries on attributes that aren't permanent 
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Parsing language                
+            </td>
+            <td>
+                You create grok patterns and regular expressions 
+            </td>
+            <td>
+                New Relic creates queries for you using the NRQL [aparse](/docs/nrql/nrql-syntax-clauses-functions/#func-aparse) function 
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Timing
+            </td>
+            <td>
+                Applied at ingest
+            </td>
+            <td>
+                Applied when you query
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Results
+            </td>
+            <td>
+                Makes permanent changes to stored log 
+            </td>
+            <td>
+                Temporarily alters your logs in the UI 
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Live tail logs
+            </td>
+            <td>
+                Live tail logs include any extracted values from ingest parsing
+            </td>
+            <td>
+                Live tail logs **don't** include any extracted values from query time parsing
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Exported logs
+            </td>
+            <td>
+                Exported logs include any extracted values from ingest parsing
+            </td>
+            <td>
+                Exported logs **don't** include any extracted values from query time parsing
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Number of attributes
+            </td>
+            <td>
+                A maximum of [255 attributes](/docs/logs/log-api/log-event-data/#events) is available at ingest (the actual number attributes you can parse at ingest depends on the nature of your logs)
+            </td>
+            <td>
+                You can parse a maximum of 32 temporary attributes across all rules for query time parsing
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## How to create a query time parsing rule [#how-to] 
+
+Here's a guide to creating query time parsing rules. The example shows how to list all the logs that contain the string `Running script`. 
+
+<Steps>
+    <Step>
+        #### Select attribute value to parse [#select-attribute]
+        You can start creating a query time parsing rule by selecting an attribute value to parse.  
+
+            1. In the log table or in the **Formatted** tab of the log details view, highlight an anchor string that contains the string you want to extract. In this case, you'd highlight `level=info msg="Running script"`. It looks like this in the logs table:
```

Comment:
Small suggestion, if we could use the term "values" for what's being extracted, that would be great.
```suggestion
            1. In the log table or in the **Formatted** tab of the log details view, highlight an anchor string that contains the values you want to extract. In this case, you'd highlight `level=info msg="Running script"`. It looks like this in the logs table:
```